ios/Podfile.lock

```diff
@@ -1141,6 +1141,6 @@ SPEC CHECKSUMS:
   Yoga: f87bd008d34c18bf990d4371200802f2866ce18c
   YogaKit: f782866e155069a2cca2517aafea43200b01fd5a
 
-PODFILE CHECKSUM: 69f576e475be6d130ee96103b131f71bfdbf8a6e
+PODFILE CHECKSUM: 054ef63292919db072e55b622ee63b87c83d39a0
 
-COCOAPODS: 1.11.3
+COCOAPODS: 1.12.1
```

Comment:
Was this expected? I'm not sure we should bump cocoapods within this PR unless absolutely necessary.

Reply:
AFAIK On `main` we are using `1.12.1` https://github.com/Expensify/App/blob/main/ios/Podfile.lock#L1237